From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nico314159 <nicolino.will@gmail.com>
Date: Tue, 13 Dec 2022 19:55:19 -0800
Subject: [PATCH] Remove-value-clamp


diff --git a/-m b/-m
new file mode 100644
index 0000000000000000000000000000000000000000..f4805c3b18a416f4beefe97332328cd7ece76cb6
--- /dev/null
+++ b/-m
@@ -0,0 +1,11 @@
+Remove value clamp
+# Please enter the commit message for your changes. Lines starting
+# with '#' will be ignored, and an empty message aborts the commit.
+#
+# On branch master
+# Your branch is ahead of 'upstream/master' by 947 commits.
+#   (use "git push" to publish your local commits)
+#
+# Changes to be committed:
+#	modified:   src/main/java/net/minecraft/world/level/Explosion.java
+#
diff --git a/src/main/java/net/minecraft/world/level/Explosion.java b/src/main/java/net/minecraft/world/level/Explosion.java
index e7ce14ab0732034107e735787354a6fb0ec90f54..00902f422abb45c05e854430b37a9e5d536e2414 100644
--- a/src/main/java/net/minecraft/world/level/Explosion.java
+++ b/src/main/java/net/minecraft/world/level/Explosion.java
@@ -86,7 +86,7 @@ public class Explosion {
         this.hitPlayers = Maps.newHashMap();
         this.level = world;
         this.source = entity;
-        this.radius = (float) Math.max(power, 0.0); // CraftBukkit - clamp bad values
+        this.radius = (float) power; // remove value clamp
         this.x = x;
         this.y = y;
         this.z = z;
@@ -136,11 +136,6 @@ public class Explosion {
     }
 
     public void explode() {
-        // CraftBukkit start
-        if (this.radius < 0.1F) {
-            return;
-        }
-        // CraftBukkit end
         this.level.gameEvent(this.source, GameEvent.EXPLODE, new Vec3(this.x, this.y, this.z));
         Set<BlockPos> set = Sets.newHashSet();
         boolean flag = true;
